.changeset/fifty-mice-play.md

```diff
@@ -0,0 +1,5 @@
+---
+'@lit-labs/react': patch
+---
+
+Boolean attributes are now removed from an HTMLElement when they are false.
```

Comment:
This needs a better description which matches the also requested clarification statement in the PR description. In particular, we need to make it clear that this issue is related to platform global boolean attributes.

Reply:
updated!